pact4s/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/provider/LeoProvider.scala

```diff
@@ -71,9 +71,13 @@ class LeoProvider extends AnyFlatSpec with BeforeAndAfterAll with PactVerifier {
   // This function composes a large switch statement based on partial functions from
   // multiple state mangers (each with their own state that they handle).
   private val providerStatesHandler: StateManagementFunction = StateManagementFunction {
-    RuntimeStateManager.handler(mockRuntimeService).orElse(AppStateManager.handler(mockAppService)).orElse { case _ =>
-      loggerIO.debug("Whoops: other state")
-    }
+    AppStateManager
+      .handler(mockAppService)
+      .orElse(DiskStateManager.handler(mockDiskService))
+      .orElse(RuntimeStateManager.handler(mockRuntimeService))
```

Comment:
I noticed the order of the changed. Curious to better understand why - thanks!

Reply:
I should have left a comment here. Thanks for asking.

I redid the order so that the state manager's are displayed in alphabetical order.